R/diffdf.R

```diff
@@ -89,9 +91,16 @@ diffdf <- function(
     strict_factor = TRUE,
     file = NULL,
     tolerance = sqrt(.Machine$double.eps),
-    scale = NULL
+    scale = NULL,
+    check_df_class = TRUE
 ) {
 
+    assertthat::assert_that(
+        is.logical(check_df_class),
```

Comment:
you can use is.flag

Reply:
there may be other instances of this. There is.number, is.string etc, now we use assertthat